v22.1/alter-default-privileges.md

```diff
@@ -32,7 +32,8 @@ Parameter | Description
 `FOR ALL ROLES` | Alter the default privileges on objects created by all users/roles.
 `GRANT ...` | Grant a default privilege or list of privileges on all objects of the specified type to a role/user, or a list of roles/users.
 `REVOKE ...` | Revoke a default privilege or list of privileges on all objects of the specified type from a role/user, or a list of roles/users.
-`IN SCHEMA qualifiable_schema_name` | This clause is a no-op. The syntax is supported for compatibility with PostgreSQL.
+`IN SCHEMA qualifiable_schema_name` | **New in v22.1:** If specified, the default privileges are altered for objects created in that schema. If an object has default privileges specified at
+the database and at the schema level, the union of the default privileges are taken.
```

Comment:
See suggestion below; also, it looks like there is an extra line break above this block.

```suggestion
the database and at the schema level, the union of the default privileges is taken.
```